src/Microsoft.TemplateEngine.Core/Util/ProcessorState.cs

```diff
@@ -14,6 +14,8 @@ public class ProcessorState : IProcessorState
     {
         private Stream _source;
         private readonly Stream _target;
+        private byte[] bom;
+        private Boolean needBom;
```

Comment:
minor: in this case `isBomNeeded` is a more c-sharpie variable name

Some discussion: https://softwareengineering.stackexchange.com/questions/232213/should-i-always-use-is-as-prefix-for-boolean-variables